### PR TITLE
Reader: New Full Post skeleton

### PR DESCRIPTION
**Pull request by blowery**
to test, you have to enable the feature flag at runtime:
`ENABLE_FEATURES=reader/refresh-2016-07 make run`

This is _really_ basic. It just loads up the new skeleton for the full post view and not much more.

This is just to get the basics in place so we can start fleshing out the new full post.

### CODE REVIEW

File: client/components/reader-full-post/index.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * Internal Dependencies
+ */
+import Main from 'components/main';
+import StickyPanel from 'components/sticky-panel';
+import Gridicon from 'components/gridicon';
+
+export default class FullPostContainer extends React.Component {
+
+	componentWillMount() {
+		document.getElementsByClassName( 'layout' )[ 0 ].classList.add( 'has-no-sidebar' );
```
Comment (mtias) [inline client/components/reader-full-post/index.jsx]: Can we do this with the `section` tree in state?

Comment (mtias) [inline client/components/reader-full-post/index.jsx]: Like this: https://github.com/Automattic/wp-calypso/blob/master/client/wordpress-com.js#L130

File: client/reader/controller.js [hunk 2]
```diff
@@ -144,6 +116,11 @@ module.exports = {
 		next();
 	},
 
+	preloadFullPost: function( context, next ) {
+		preload( 'reader-full-post' );
```
Comment (mtias) [inline client/reader/controller.js]: Nice :)

**Comment by blowery** [comment]
@bluefuton could use some 👀

File: client/components/reader-full-post/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Reader Full Post
+
+The "new" reader full post component
```
Comment (bluefuton) [inline client/components/reader-full-post/README.md]: Worth adding a tiny bit of blurb about what the new full post view does differently?

**Comment by bluefuton** [comment]
Checked with and without the feature flag enabled. 

Looks good to 🚢

**Pull request merged**